plugins/PostProcessingPlugin/scripts/ExampleScript.md

```diff
@@ -1,3 +0,0 @@
-A good example script is SearchAndReplace.py.
-If you have any questions please ask them at: 
-https://github.com/Ultimaker/Cura/issues 
```

Comment:
Although this file is a bit non-sense, I don't think I should be removed in this PR.